include/dgl/runtime/shared_mem.h

```diff
@@ -27,21 +27,27 @@ class SharedMemory {
    * and will be responsible for deleting it when the object is destroyed.
    */
   bool own;
+
+  /* \brief the file descripter of the shared memory. */
+  int fd;
+  /* \brief the address of the shared memory. */
+  void *ptr;
```

Comment:
Will you free them in the destructor?

Reply:
SharedMem will be destructed at:
https://github.com/dmlc/dgl/blob/2190c39d674f76c65db9ee8da7b43d3021f19c29/src/runtime/shared_mem.cc#L28

Raw pointer will be destructed at:
https://github.com/VoVAllen/dgl/blob/72514729dd2cc2cdfc6c52812dff7283508035f9/src/graph/serialize/zerocopy_serializer.cc#L22